### PR TITLE
Sample story for component that depends on Apollo

### PR DESCRIPTION
**Pull request by eddieferrer**
* Story clean  up
* Added apollo mocking for global promo component
* Added svg sprite generation for storybook

VUE-196

### CODE REVIEW

File: .storybook/stories/GlobalPromoBanner.stories.js [hunk 1]
```diff
@@ -0,0 +1,69 @@
+import Vue from 'vue'
+import StoryRouter from 'storybook-vue-router';
+import GlobalPromoContentful from '@/components/WwwFrame/PromotionalBanner/GlobalPromotionalBannerContentful';
+
+// import plugins
+import kivaPlugins from '@/plugins';
+Vue.use(kivaPlugins)
+
+let mockedContentfulAPIResponse = {
```
Comment (ryan-ludwig) [inline .storybook/stories/GlobalPromoBanner.stories.js]: Nitpick, should this be a const instead?

Comment (eddieferrer) [reply .storybook/stories/GlobalPromoBanner.stories.js]: 👍

**Comment by mcstover** [comment]
Hmmm. I think this speaks to us needed to decouple the datasource and the markup for the global promo banner. We should setup global promo banner so it accepts an icon id, link, copy and click track data.

I think there is something useful in being able to render a component like this from mocked data but even then that could be a data provider for the base version of global promo banner.

**Comment by eddieferrer** [comment]
> Hmmm. I think this speaks to us needed to decouple the datasource and the markup for the global promo banner. We should setup global promo banner so it accepts an icon id, link, copy and click track data.
> 
> I think there is something useful in being able to render a component like this from mocked data but even then that could be a data provider for the base version of global promo banner.

Yeah I agree, the component that is decoupled would be the <generic-promo-banner>, there might not be too much value in making a story for this component as opposed to the <generic-promo-banner> component...

**Comment by eddieferrer** [comment]
Added some additional improvements: 
* src/components/WwwFrame/PromotionalBanner/GlobalPromotionalBannerContentful.vue is just a wrapper now that gets contentful data and passes it to the generic promo banner
* generic promo banner is dumber, just takes in data now. 
* kept the contentful banner story as an example of how to do a story for component that depends on this.apollo.query with fake content
* added more useful story for generic promo banner with knobs for text/icon/link

**Pull request merged**